src/pages/Pools/LiquidityMiningCampaign/index.tsx

```diff
@@ -146,7 +146,7 @@ export default function LiquidityMiningCampaign({
                 }}
               >
                 <Text fontWeight={700} fontSize={12}>
-                  {isSingleSidedCampaign ? 'GET SWPR' : 'ADD LIQUIDITY'}
+                  {isSingleSidedCampaign ? `GET ${token0?.symbol}` : 'ADD LIQUIDITY'}
```

Comment:
```suggestion
                  {isSingleSidedCampaign ? `GET ${token0?.symbol ?? 'TOKEN' }` : 'ADD LIQUIDITY'}
```

Since token is optional. But its ok 😄